### ISSUE TITLE
Is this still being maintained?

### ISSUE THREAD

**Issue by Clickative** [markfinger/python-react#92]

**Comment by markfinger** [comment]
Yes. It's a mature system that's hasn't evolved for a while as its largely feature complete. 

Occasionally I'll make some minor changes to reflect React's API changes, but that's about it. Sporadically someone opens a PR to make a small change or bolt on something.

Is there a particular problem that you want attention on?

**Comment by Clickative** [comment]
I am a little concerned about the fact that the last release is about 2 years old. I was hoping to use this as part of a large app and I am not very sure about how this would be scaling and if it is a good development practice to use it.

**Comment by Clickative** [comment]
Also, I believe that the documentation for this project is rather lacking considering its potential use case.

**Comment by markfinger** [comment]
> I am a little concerned about the fact that the last release is about 2 years old

I think you might misunderstand the scale of this lib. It's just some wiring that talks to a render server and shuttles data between them. If for some reason you need to reimplement this project, it'd take you an afternoon. 

There are also a variety of others implementations, for example: https://github.com/mic159/react-render.

> I was hoping to use this as part of a large app and I am not very sure about how this would be scaling and if it is a good development practice to use it.

TBH, I'd hope that good development practice is to not use SSR unless you really need it. It complicates a lot of things, including your codebase and infrastructure. Most of the time you can get a comparable experience + SEO boost by simply inlining the data that you'd be fetching asynchronously from your API.

> Also, I believe that the documentation for this project is rather lacking considering its potential use case.

You're welcome to ask questions.
